condor_watch_q.py

```diff
@@ -519,9 +519,20 @@ def row_data_from_job_state(clusters, summary):
 
             if job_state in ACTIVE_STATES:
                 active_job_ids.append("{}.{}".format(cluster.cluster_id, proc_id))
+                
+    maxmin_job = []
```

Comment:
I'd prefer if we did the max-min stuff in a separate PR; this one already has two separate changes in it (summary and - instead of 0)